src/components/app/details/cicdHistory/Artifacts.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { showError, copyToClipboard } from '../../../common'
+import { toast } from 'react-toastify'
+import { useParams } from 'react-router'
+import { ReactComponent as CopyIcon } from '../../../../assets/icons/ic-copy.svg'
+import { ReactComponent as Download } from '../../../../assets/icons/ic-download.svg'
+import { ReactComponent as MechanicalOperation } from '../../../../assets/img/ic-mechanical-operation.svg'
+import docker from '../../../../assets/icons/misc/docker.svg'
+import folder from '../../../../assets/icons/ic-folder.svg'
+import Tippy from '@tippyjs/react'
+import EmptyState from '../../../EmptyState/EmptyState'
+import { EmptyView } from './History.components'
+import '../cIDetails/ciDetails.scss'
+import { ArtifactType, CopyTippyWithTextType } from './types'
+export default function Artifacts({ status, artifact, blobStorageEnabled, getArtifactPromise }: ArtifactType) {
+    const { buildId, triggerId } = useParams<{ buildId: string; triggerId: string }>()
+    async function handleArtifact(e) {
+        try {
+            const response = await getArtifactPromise()
+            const b = await (response as any).blob()
+            let a = document.createElement('a')
+            let url = URL.createObjectURL(b)
```

Comment:
We can use `URL.createObjectURL(b)` directly in next line

Reply:
done